recipes/labbench/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "labbench" %}
+{% set version = "0.37.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/labbench-{{ version }}.tar.gz
+  sha256: 922a38307906bfca192845ad6b40b65bdf3db8b2ddc35ac133b36072ecf249fe
+
+build:
+  skip: True  # [py >= 313]
+  skip: True  # [py < 39]
+  entry_points:
+    - labbench = labbench.cli.__main__:do_cli
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - hatchling
+    - conda-verify
+    - typing-extensions
+    - future
+  run:
+    - python
+    - psutil >=5.0
+    - validators >=0.20.0
+    - pip
```

Comment:
`pip` is normally not needed at runtime
```suggestion
```

Reply:
you're right to point this out, but labbench actually does use pip at runtime in order logging installed modules